src/mscorlib/src/System/Runtime/Intrinsics/X86/Avx.cs

```diff
@@ -117,7 +117,16 @@ public static class Avx
         /// __m256d _mm256_cmp_pd (__m256d a, __m256d b, const int imm8)
         /// </summary>
         public static Vector256<double> Compare(Vector256<double> left, Vector256<double> right, FloatComparisonMode mode) => Compare(left, right, mode);
-        
+
+        /// <summary>
+        /// __m128 _mm_cmp_ss (__m128 a, __m128 b, const int imm8)
+        /// </summary>
+        public static Vector128<float> CompareScalar(Vector128<float> left, Vector128<float> right, FloatComparisonMode mode) => CompareScalar(left, right, mode);
```

Comment:
Was wondering why we have `FloatComparisonMode` here, but not in the `Sse`/`Sse2` files?